addons/account/views/account_portal_templates.xml

```diff
@@ -104,7 +104,7 @@
         <xpath expr="//div[hasclass('o_portal_sidebar')]" position="inside">
             <t t-set="o_portal_fullwidth_alert" groups="sales_team.group_sale_salesman,account.group_account_invoice,account.group_account_readonly">
                 <t t-call="portal.portal_back_in_edit_mode">
-                    <t t-set="backend_url" t-value="'/web#model=%s&amp;id=%s&amp;action=%s&amp;view_type=form' % (invoice._name, invoice.id, invoice.env.ref('account.action_move_out_invoice_type').id)"/>
+                    <t t-set="backend_url" t-value="'/odoo/action-%s/%s' % (invoice.env.ref('account.action_move_out_invoice_type').id, invoice.id)"/>
```

Comment:
I only just learnt this from the diff below...
Could it be
```suggestion
                    <t t-set="backend_url" t-value="'/odoo/action-account.action_move_out_invoice_type/%s' % invoice.id)"/>
```

Reply:
Hello @h4818 and @jpp-odoo 

Changes have been made, please check it.
Thanks :slightly_smiling_face: